ios/FluentUI/Vnext/Shimmer/ShimmerLinesView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
```

Comment:
something is weird. I would have expected git to pick up this was a move from the "old location"

Reply:
wait... what's the standnard here? why are we moving controls to vnext folder? i thought we only did that if we need side by side?

Reply:
Hm yeah, in GitHub Desktop, it showed up as a move.

It's my understanding that tokenized controls get moved to the vNext folder once they're tokenized (Button is the only control in the vNext folder that I see with a legacy file/folder maintained)

Reply:
I just looked at the first commit and it shows up as a move:
<img width="286" alt="image" src="https://user-images.githubusercontent.com/31874971/173163817-11ddb8c7-5a4e-47fd-9b25-f4ca3e41d7ef.png">